src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/GameResults/ResultsValidate.js

```diff
@@ -295,6 +295,9 @@
 
   $(document).ready(function () {
     privates.formLoaded = $("#createForm").serialize();
+    $(function () {
```

Comment:
Why to wrap into function?

Add comment explaining purpose

Reply:
it is not working without that